components/org.wso2.carbon.identity.api.server.email.template/org.wso2.carbon.identity.rest.api.server.email.template.v1/pom.xml

```diff
@@ -169,6 +169,22 @@
                 </exclusion>
             </exclusions>
         </dependency>
+        <dependency>
+            <groupId>org.wso2.carbon.identity.server.api</groupId>
+            <artifactId>org.wso2.carbon.identity.api.server.common</artifactId>
+            <scope>provided</scope>
+            <version>${project.version}</version>
```

Comment:
No need to have a version here. Define the version in the dependency management section in the root pom of the project. Fix the same in all places.